concent_api/verifier/tasks.py

```diff
@@ -40,7 +40,7 @@ def blender_verification_order(
     result_package_path: str,
     result_size: int,
     result_package_hash: str,
-    output_format: str,
+    output_format: IntEnum,
```

Comment:
Have you tested if `IntEnum` is serializable by celery ? Testing this with integrated test is not enough, even running task as function is not enough, you have to test it manually in Celery queue.

Reply:
Issue describes `IntEnum` is serializable and it is tested, but not like you have written. I will test it like this as well